docs/get-started/lotus/bootstrappers.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Running Bootstrapper Nodes
+description: 'Running your own bootstrapper nodes'
+breadcrumb: 'Bootstrapper nodes'
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }}
+
+### Bootstrappers
+
+Lotus daemons may be configured to act as network bootstrappers. Bootstrappers act as the initial point of
+contact for other lotus daemons to find peers.
+
+::: tip
+Unless you are running a private test network, it is not usuaally necessary for to run a network bootstrapper node.
+:::
+
+### Configuring a Bootstrapper node
+
+Because bootstrapper nodes handle somewhat different traffic than a typical lotus daemon, we recommend using
+the following configuration. This configuration will setup the node to act as a bootstrapper for 
+
+```toml
+  [API]
+    ListenAddress = "/ip4/0.0.0.0/tcp/1234/http"
+  [Libp2p]
+    ListenAddresses = ["/ip4/0.0.0.0/tcp/1347"]
+    ConnMgrLow = 400
+    ConnMgrHigh = 500
+    ConnMgrGrace = "5m0s"
+  [Pubsub]
+    IPColocationWhitelist = ["x.x.x.x/x"]
+    Bootstrapper = true               # This line is required at time of writing to act as Pubsub Bootstrapper
+
+```
+
+### Starting a Bootstrapper node
+
+Start the lotus daemon with the bootstrapper profile. `--profile=bootstraper` overrides configuration defaults
+to enable bootstrapper mode for libp2p and pubsub subsystems.
+
+```bash
+lotus daemon --profile=bootstrapper
+```
+
+### Operating a Bootstrapper node
+Peer your bootstrapper node with other bootstrappers on the same network.
+
+```bash
+lotus net peers
+lotus net connect <peer_multiaddr>
+```
```

Comment:
Regardless of the profile, the node will still bootstrap, so I'm not sure this is really needed, maybe it can be changed to say "How to connect to a node" and maybe merged with the section below?